src/NUnitFramework/tests/Assertions/AssertThatTests.cs

```diff
@@ -655,6 +664,8 @@ public void TestPropertyFailureSecondLevel()
              */
         }
 
+        private record Record(string Name, int[] Collection);
```

Comment:
I would like an example of a `record` where the user overrides `Equals`.
E.g. To use Case insensitive comparison of `Name`.
```csharp
private sealed record Person(string Name)
{
    public bool Equals(Person? other)
    {
        return string.Equals(Name, other?.Name, System.StringComparison.OrdinalIgnoreCase);
    }

    public override int GetHashCode()
    {
        return Name.GetHashCode(System.StringComparison.OrdinalIgnoreCase);
    }
}
```